### ISSUE TITLE
Needs Specific Java Runtime?

### ISSUE THREAD

**Issue by MonfGeiger** [Wizelf402/Permafrost#74]
I discovered this modpack from (i kid you not) a fanfic story on fanficton.net, so i decided to download it via the FTB App and use MultiMC to import it over (im more of an AIO (All In One) type person instead of 3+ launchers for different and/or launcher exclusive modpacks, no offense meant). Thing is, it only got the config folder, so i did the good ol copy/paste and rechecked the instance settings to make sure everything was kosher, ran it, and it crashed.

I later found out from the MultiMC support team that the forge version its running (36.2.23) doesnt like java 8u351 which i use as default, so i grabbed an archived .zip zulu build of 8u312 and it runs fine. I wanted some further clarification: was this modpack **always** meant to run off a specific forge & java version, or could i bump up the forge number and keep my 8u351 instead? I already have 3 java's installed (8u351, jdk 17, and jdk19), adding a fourth feels a bit excessive

**Comment by NevadaActual** [comment]
As far as I know, only specific Java versions cooperate with the mods we have in the pack, due to deprecation and Forge shenanigans. If you can get it running by bumping up the Forge version, then there shouldn't be any further problems. I've been planning to make the so-called "final" version of the pack with a pre-packaged java installation, just to avoid problems like this, but I haven't gotten around to it just yet.

If you only got the config folder, then I'll need to check the pack to make sure something isn't screwy, there are a lot of custom resources we use in the pack.

TLDR: It's "intended" to run with a specific java version, in the sense that further versions or versions before it crash due to reasons outside our control. *Whatever* lets you run it is intended.

**Comment by NevadaActual** [comment]
Yes, it does appear the download is working. I'd suggest going through this link: 
https://www.curseforge.com/minecraft/modpacks/permafrost-eots/files
This will have all of the graphics, as well as the necessary configurations needed to run things properly. Let me know if I can help you with anymore issues!

(I appreciate someone wrote fanfiction about the pack, I'll be sure to let Vixie know.)

**Comment by MonfGeiger** [comment]
For the bug that was encountered, phit over at the MultiMC launcher github said the [following](https://github.com/MultiMC/Launcher/issues/5113#issuecomment-1492879164)

As for the fanfic, [here](https://www.fanfiction.net/s/14193149/1/Thawing-Out)

Also, last question, what's the overrides.zip in the modpack for?

**Comment by NevadaActual** [comment]
The "overrides" zip is essentially anything non-vanilla about the configs, the graphics, or the localizations. Since curseforge already hosts the mods on their website, they don't keep them with any main modpack download, just to ease the development process since it saves the developers from having to reupload *all* the mods *every* time. Serverpacks are different, because of reasons mostly involving some mods being clientside only, etc, but for singleplayer/client installations they download the overrides to override everything after the launcher downloads all the mods.

Sweet, I'll refer back to that bug when I make the prepackaged install later. Is there any other issues, or should I close this issue?

**Comment by MonfGeiger** [comment]
So i wouldnt need to extract the overrides zip onto the modpack already if i imported it to MultiMC, or i guess it wouldnt really matter at that point then

either way, all my questions are solved then i guess

**Comment by NevadaActual** [comment]
If things are loading, like the custom texture pack, menu, and the starting items are also loading? Then you should be set, yet.
